src/main/java/bio/terra/cda/app/controller/QueryApiController.java

```diff
@@ -89,4 +94,25 @@ private String createNextUrl(String jobId, int offset, int limit) {
 
     return sendQuery(querySql, dryRun);
   }
+
+  @Override
+  public ResponseEntity<QueryResponseData> uniqueValues(String version, @Valid String body) {
+
+    String bqTable = "gdc-bq-sample";
+    Map<String, String> tableParts = QueryTranslator.parseTableName(body);
+    // SELECT * FROM UNNEST (SELECT DISTINCT p.column FROM qualifiedTABLE as p);
+    String querySql =
+        "SELECT DISTINCT "
+            + tableParts.get("columnName")
+            + " FROM "
+            + bqTable
+            + "."
+            + tableParts.get("tableName");
+    logger.info("uniqueValues: " + querySql);
+
+    ResponseEntity<QueryCreatedData> response = sendQuery(querySql, false);
+    var query_id = response.getBody().getQueryId();
+
+    return query(query_id, 0, 1000);
```

Comment:
Like the other query APIs, this should return `sendQuery(querySql, false)` directly. The API user will need to poll `query` for the result.

Reply:
Alternatively, you could make this return the list of names and not return any of the query data at all. You'd have to change this code to loop over `query()` until it's complete, and then extract out the response body so it just contains a list of strings. If you want to do that, it would be better to do it in `QueryService` as its own method.

Reply:
Now matches the other api calls.